src/games/strategy/engine/framework/startup/mc/ClientModel.java

```diff
@@ -335,35 +335,38 @@ private void startGameInNewThread(final byte[] gameData, final Map<String, INode
     final Set<IGamePlayer> playerSet = data.getGameLoader().createPlayers(playerMapping);
     final Messengers messengers = new Messengers(m_messenger, m_remoteMessenger, m_channelMessenger);
     m_game = new ClientGame(data, playerSet, players, messengers);
-    final Thread t = new Thread("Client Game Launcher") {
-      @Override
-      public void run() {
-        SwingUtilities.invokeLater(() -> JOptionPane.getFrameForComponent(m_ui).setVisible(false));
-        try {
-          // game will be null if we loose the connection
-          if (m_game != null) {
-            try {
-              data.getGameLoader().startGame(m_game, playerSet, false);
-              data.testLocksOnRead();
-            } catch (final Exception e) {
-              ClientLogger.logQuietly(e);
+    new Thread(() -> {
```

Comment:
This is not the actual line... deleted the line which call startGameIn new thread from a new thread